.circleci/config.yml

```diff
@@ -13,14 +13,18 @@ jobs:
 
     steps:
       - checkout
-
       # Download and cache dependencies
+
       - restore_cache:
           keys:
           - v1-dependencies-{{ checksum "package.json" }}
           # fallback to using the latest cache if no exact match is found
           - v1-dependencies-
 
+      - run:
+          name: update-npm
+          command: 'sudo npm install -g npm@latest'
+
```

Comment:
Please update npm *before* restoring the cache. I think this may be causing the build failure.